src/prefect/cli/global_concurrency_limit.py

```diff
@@ -297,3 +316,87 @@ async def update_global_concurrency_limit(
                 )
 
     exit_with_success(f"Updated global concurrency limit with name {name!r}.")
+
+
+@global_concurrency_limit_app.command("create")
+async def create_global_concurrency_limit(
+    name: str = typer.Argument(
+        ..., help="The name of the global concurrency limit to create."
+    ),
+    limit: int = typer.Option(
+        ..., "--limit", "-l", help="The limit of the global concurrency limit."
+    ),
+    disable: Optional[bool] = typer.Option(
+        None, "--disable", help="Create an inactive global concurrency limit."
+    ),
+    active_slots: Optional[int] = typer.Option(
+        0, "--active-slots", help="The number of active slots."
+    ),
+    slot_decay_per_second: Optional[float] = typer.Option(
+        0.0, "--slot-decay-per-second", help="The slot decay per second."
+    ),
+):
+    """
+    Create a global concurrency limit.
+
+    Arguments:
+
+        name (str): The name of the global concurrency limit to create.
+
+        limit (int): The limit of the global concurrency limit.
+
+        disable (Optional[bool]): Create an inactive global concurrency limit.
+
+        active_slots (Optional[int]): The number of active slots.
+
+        slot_decay_per_second (Optional[float]): The slot decay per second.
+
+    Examples:
+
+        $ prefect global-concurrency-limit create my-gcl --limit 10
+
+        $ prefect gcl create my-gcl --limit 5 --active-slots 3
+
+        $ prefect gcl create my-gcl --limit 5 --active-slots 3 --slot-decay-per-second 0.5
+
+        $ prefect gcl create my-gcl --limit 5 --inactive
+    """
+    async with get_client() as client:
+        try:
+            await client.read_global_concurrency_limit_by_name(name=name)
+        except ObjectNotFound:
+            pass
+        else:
+            exit_with_error(
+                f"Global concurrency limit {name!r} already exists. Please try creating with a different name."
+            )
+
+    try:
+        gcl = GlobalConcurrencyLimitCreate(
+            name=name,
+            limit=limit,
+            active=False if disable else True,
+            active_slots=active_slots,
+            slot_decay_per_second=slot_decay_per_second,
+        )
+
+    except ValidationError as exc:
+        exit_with_error(f"Invalid arguments provided: {exc}")
+    except Exception as exc:
+        exit_with_error(f"Error creating global concurrency limit: {exc}")
+
+    async with get_client() as client:
+        try:
+            gcl_id = await client.create_global_concurrency_limit(concurrency_limit=gcl)
+        except PrefectHTTPStatusError as exc:
+            # We shouldn't hit this as we validate the model above
+            if exc.response.status_code == 422:
+                parsed_response = exc.response.json()
+
+                exc = parsed_response["exception_detail"][0]["msg"]
```

Comment:
Since we wouldn't expect a 422, because as you've said we validate the model before the request, what about just making this generically handle any http status error?

```suggestion
            parsed_response = exc.response.json()
            exc = parsed_response["exception_detail"][0]["msg"]
```

Reply:
good thinking